lib/internal/cluster/master.js

```diff
@@ -70,9 +70,9 @@ cluster.setupMaster = function(options) {
   assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
          `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
 
-  const hasDebugArg = process.execArgv.some((argv) => {
-    return /^(--debug|--debug-brk)(=\d+)?$/.test(argv);
-  });
+  const hasDebugArg = process.execArgv.some(
+    (argv) => /^(--debug|--debug-brk)(=\d+)?$/.test(argv)
```

Comment:
(Although this particular line is probably a low priority to tweak since it seems a likely candidate to go away on master when we update to a V8 version that no longer supports the JSON debugging protocol.)